seahorn/trusty/user/base/lib/tipc/proof/tipc_send1/main.c

```diff
@@ -0,0 +1,35 @@
+#include <seahorn/seahorn.h>
+#include <lib/tipc/tipc.h>
+#include <trusty_ipc.h>
+#include <stdlib.h>
+
+extern handle_t nd_handle(void);
+extern int nd_size_t(void);
+//extern void* can_fail_malloc(int);
+extern int msg_send_called;
+
+/** Test harness for tpic_send1 */
+int main(void) {
+
+ // msg_is_send_called = 0;
+
+  /* assumptions */
+ msg_send_called = 0;
```

Comment:
I now understand why the variable is declared in the header file.
Would be cleaner to expose a method to reset the state.